### PR TITLE
New scroll example and reduced compile time for lib using dev-dependencies

### PR DESCRIPTION
**Pull request by BlackPhlox**
Don't merge quite yet, I want to hear your opinion. 
Also, for some reason changing the FOV (the [scroll.rs](https://github.com/sburris0/bevy_flycam/compare/master...BlackPhlox:master#diff-49c1807c54650a4bbb81535c74019611e509e6555cfb8742a31906af82c82b5aR89)) of the camera doesn't work even though it does change, maybe the fov is only read on initialization?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,11 @@ There are a few notable differences from [bevy_fly_camera](https://github.com/mc
 * Single-line setup
 * A tiny bit faster?
 
+Comparing [bevy_4x_camera](https://github.com/SarthakSingh31/bevy_4x_camera) :
```
Comment (sburris0) [inline README.md:24]: I don't think this comparison is necessary, it's a different type of camera and the choice should be somewhat obvious depending on your game's goals.

File: README.md [hunk 2]
```diff
@@ -55,7 +60,7 @@ fn main() {
 }
 ```
 
-Alternativly you can see the example `basic.rs` located in the examples folder.
+Alternativly you can see the example `basic.rs` or `scroll.rs` located in the examples folder.
```
Comment (sburris0) [inline README.md:63]: Alternatively

**Review by sburris0 — CHANGES_REQUESTED** [review]
Looks good to me, more examples are always good.
I only have two small feedback items

File: README.md (hunk 2 above)
Comment (BlackPhlox) [reply README.md]: Got my first mechanical keyboard this week and still adjusting to it. However, I've now installed a spell checker extension so it shouldn't happen again, thanks for the feedback 😊

**Comment by sburris0** [comment]
Looks good to me, thanks for contributing!

**Pull request merged**